### ISSUE TITLE
[Suggestion] Toggle the hud control labels

### ISSUE THREAD

**Issue by DrizztCode** [tsunamayo/Starship-EVO#5601]
hi - i can't find the option, so if it doesn't exist i am suggesting to add a way to toggle off the labels saying which buttons control the hud elements

thanks :-)

**Comment by tsunamayo** [comment]
Yes there is no option for that sorry. I will see if I can add this.

**Comment by DrizztCode** [comment]
thanks - obviously just a very little thing - but i saw it so i said it :-)
